finagle-netty4/src/main/scala/com/twitter/finagle/netty4/channel/ServerBridge.scala

```diff
@@ -15,6 +17,17 @@ private[netty4] class ServerBridge[In, Out](
 
   def initChannel(ch: Channel): Unit = {
     val transport: Transport[In, Out] = transportFac(ch)
-    serveTransport(transport)
```

Comment:
@vkostyukov I think I figured it out - we are using backpressure, so channel's auto read is set to `false`.  Means `transportFac(ch)` cannot be delayed